userspace/falco/configuration.cpp

```diff
@@ -148,7 +148,23 @@ void falco_configuration::init(string conf_filename, list<string> &cmdline_optio
 		m_outputs.push_back(http_output);
 	}
 
-	if (m_outputs.size() == 0)
+	m_grpc_enabled = m_config->get_scalar<bool>("grpc", "enabled", false);
+	m_grpc_bind_address = m_config->get_scalar<string>("grpc", "bind_address", "0.0.0.0:5060");
+	m_grpc_threadiness = m_config->get_scalar<uint32_t>("grpc", "threadiness", 8); // todo > limit it to avoid overshubscription? std::thread::hardware_concurrency()
+	// todo(fntlnz,leodido) > chose correct paths
+	m_grpc_private_key = m_config->get_scalar<string>("grpc", "private_key", "/etc/falco/certs/server.key");
+	m_grpc_cert_chain = m_config->get_scalar<string>("grpc", "cert_chain", "/etc/falco/certs/server.crt");
+	m_grpc_root_certs = m_config->get_scalar<string>("grpc", "root_certs", "/etc/falco/certs/ca.crt");
```

Comment:
I think any hard coded strings should be `--flags`?

Reply:
These are defaults that (should) match the default values we have in the `falco.yaml` config file.